iothub/device/samples/how to guides/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -351,11 +313,6 @@ private async Task SendMessagesAsync(CancellationToken cancellationToken)
 
         private async Task ReceiveMessagesAsync(CancellationToken cancellationToken)
         {
-            var c2dReceiveExceptionsToBeIgnored = new Dictionary<Type, string>(_exceptionsToBeIgnored)
```

Comment:
Actually, we shouldn't be retrying CompleteAsync on this exception. It should gracefully fail, and the message will be redelivered.